code/game/gamemodes/blob/theblob.dm

```diff
@@ -196,6 +198,11 @@
 		color = a_color
 	return
 
+/obj/effect/blob/proc/update_desc(var/a_desc=null)

```

Comment:
Is all this even necessary? Can't it just read the reagent off in the examine proc?